lib/pbench/server/cache_manager.py

```diff
@@ -344,6 +380,7 @@ def unpack(self):
             shutil.rmtree(self.cache, ignore_errors=True)
             raise
         self.unpacked = self.cache / self.name
+        self.cache_map(self.unpacked)
```

Comment:
You should also set `self.cache_map = None` in `uncache`, below. In the next step forward, we'll be enabling dynamically managing individual unpacked files, and we'll only use `uncache` during `delete` to remove the cache map itself (along, of course, with any cached files we're currently managing).